images/linux/scripts/tests/Databases.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+Describe "MongoDB" {
+    It "<ToolName>" -TestCases @(
+        @{ ToolName = "mongo" }
+        @{ ToolName = "mongod" }
+    ) {
+        "$ToolName --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "PostgreSQL" {
+
+    It "PostgreSQL Service" {
+        "sudo systemctl start postgresql" | Should -ReturnZeroExitCode
+        (Get-CommandResult "pg_isready").Output | Should -Contain "/var/run/postgresql:5432 - accepting connections"
```

Comment:
Are you sure that `Should -Contain` is correct operator here?
```
Asserts that the collection contains value specified using PowerShell's -contains operator.
```

Reply:
I guess yes

Reply:
But `(Get-CommandResult "pg_isready").Output` is always the single line. See https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/helpers/Common.Helpers.psm1#L11

So you are trying to find element in array that contains single element always. 
I could be wrong but it is equal to `(Get-CommandResult "pg_isready").Output | Should -Be "/var/run/postgresql:5432 - accepting connections"`
 Should it be -BeMatch or -BeLike?